packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -200,8 +210,12 @@ export async function run(context: $TSContext, resourceDefinition: $TSObject) {
         await deploymentManager.deploy(deploymentStateManager);
 
         // delete the intermidiate states as it is ephemeral
-        if (stateFolder && fs.existsSync(stateFolder)) {
-          fs.removeSync(stateFolder);
+        if (stateFolder.local && fs.existsSync(stateFolder.local)) {
```

Comment:
Instead of calling `fs.existsSync()`, can you call `fs.removeSync()` and handle any errors.

Reply:
printing an error as a result of this though I don't believe this should stop the deploy.